pkg/proxy/nftables/proxier.go

```diff
@@ -1136,6 +1140,21 @@ func (s *nftElementStorage) cleanupLeftoverKeys(tx *knftables.Transaction) {
 	s.resetLeftoverKeys()
 }
 
+// logFailure logs the transaction and the full table with a rate limit.
+func (proxier *Proxier) logFailure(tx *knftables.Transaction) {
+	if klogV7 := klog.V(4); klogV7.Enabled() && proxier.logRateLimiter.Allow() {
```

Comment:
I think you mean `klogV4` here, ... either one of the things is wrong or the level or the name of the var :)

being ratelimited to one log per hour and since only runs under a transaction error I'm fine with level4